packages/web/src/pages/upload-page/components/SubmitRowAnchored.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  HarmonyButton,
+  HarmonyButtonSize,
+  HarmonyButtonType,
+  IconUpload
+} from '@audius/stems'
+
+import styles from './SubmitRowAnchored.module.css'
+
+const messages = {
+  complete: 'Complete Upload'
+}
+
+export const SubmitRowAnchored = () => {
```

Comment:
AnchoredSubmitRow?

Reply:
I tend to prefer leading with the primary purpose descriptor rather than the modifier.

Though I could see an argument that the differentiator and primary purpose here is the Anchor 🤔

Reply:
i could see either way, my examples are Text -> UserGeneratedText, Image -> ProfileImage -> EditProfileImage, basically composing up from the base component, with the additional context being added in the beginning of the component. maybe a chance for us all to norm on a pattern